lib/crypto.js

```diff
@@ -15,11 +15,19 @@ exports.boxEncrypt = boxEncrypt;
 exports.boxDecrypt = boxDecrypt;
 exports.sha256 = sha256;
 
-var _crypto = require('crypto');
+var _randombytes = require('randombytes');
```

Comment:
can we remove lib/ from git and publish untranspiled (2 separate questions)?

Reply:
Ryan mentioned on a call that the codebase uses flow types so we need to transpile, but maybe we can reduce what transpiling is doing to just strip flowtypes?

Reply:
Yeah, this needs to be rethought; just fixing one thing at a time here.